proto/substrait/algebra.proto

```diff
@@ -1059,10 +1059,19 @@ message Expression {
     message PrecisionTimestamp {
       // Sub-second precision, 0 means the value given is in seconds, 3 is milliseconds, 6 microseconds, 9 is nanoseconds, 12 is picoseconds
       int32 precision = 1;
-      // Time passed since 1970-01-01 00:00:00.000000 in UTC for PrecisionTimestampTZ and unspecified timezone for PrecisionTimestamp
+      // Time passed since 1970-01-01 00:00:00.000000 an unspecified timezone.
       int64 value = 2;
     }
 
+    message PrecisionTimestampTz {
+      // Sub-second precision, 0 means the value given is in seconds, 3 is milliseconds, 6 microseconds, 9 is nanoseconds, 12 is picoseconds
+      int32 precision = 1;
+      // Time passed since 1970-01-01 00:00:00.000000 in the specified timezone.
```

Comment:
I specifically use the provided timezone since conversion to UTC and back can lose clarity due to daylight savings time.

Reply:
There is no loss of clarity when talking about _timestamps_, those are always well-defined, since they indicate an absolute point on a timeline (with possibly some nuances around how leap seconds are handled..).

What is the reason why a timezone would be needed here?